tests/test_schema.py

```diff
@@ -39,3 +51,78 @@ def test_schema(session, schema):
     assert result.data["people"]["result"][0]["name"] == "Person 1"
     assert result.data["cars"]["result"][0]["name"] == "Car 1"
     assert result.data["houses"]["result"][0]["name"] == "House 1"
+
+
+def test_page(session, schema):
```

Comment:
This should use [`pytest.mark.parametrize`](https://docs.pytest.org/en/latest/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions).